examples/mysql-atomic-batch-execute-operation/mysql_atomic_batch_execute_operation.bal

```diff
@@ -36,70 +37,67 @@ public function main() returns error? {
     ];
 
     // Creates a batch parameterized query.
```

Comment:
```suggestion
    // Creates a batch-parameterized query.
```

Reply:
Fixed in [4621ad1](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/4621ad1b1e00a60859c191de19de2041e0516f79)